tools/sdk-generation-pipeline/packages/sdk-generation-cli/src/generateResultCli.ts

```diff
@@ -1,24 +1,33 @@
 #!/usr/bin/env node
-import * as fs from 'fs';
-import { createTaskResult, AzureSDKTaskName, TaskResult, TaskOutput, LogFilter, logger } from '@azure-tools/sdk-generation-lib';
+import * as fs from "fs";
+import {
+    createTaskResult,
+    AzureSDKTaskName,
+    TaskResult,
+    TaskOutput,
+    LogFilter,
+    logger,
+} from "@azure-tools/sdk-generation-lib";
 
 function printHelp() {
-    console.log('usage: generateResult --pipelineBuildId --logfile --taskname --resultOutputPath');
-    console.log('                      [--taskOutput] [--logFilter]\n');
-    console.log('taskname: must be one of [Init, GenerateAndBuild, MockTest, LiveTest]');
+    console.log('usage: generateResult --pipelineBuildId --logfile --taskName --resultOutputPath');
+    console.log('                      --taskResultFile [--taskOutput] [--logFilter]\n');
+    console.log('taskName: must be one of [Init, GenerateAndBuild, MockTest, LiveTest]');
 }
 
 async function main() {
     const args = parseArgs(process.argv);
     const pipelineBuildId = args['pipelineBuildId'];
     const logfile = args['logfile'];
     const logFilterStr = args['logFilter'];
-    const taskname = args['taskname'];
+    const taskName = args['taskname'];
     const exeResult = args['taskExeResult']
     const taskOutput = args['taskOutput'];
     const resultOutputPath = args['resultOutputPath'];
+    const taskResultFile = args['taskResultFile'];
```

Comment:
when run docker finished, it will generate /tmp/output/taskResults.json file, just like 
{
  "init": "success",
  "generateAndBuild": "success"
}
generateResultCli use it to determine whether each task success.